mirrord/kube/src/api/kubernetes.rs

```diff
@@ -202,14 +203,15 @@ impl KubernetesAPI {
     pub async fn create_agent<P>(
         &self,
         progress: &mut P,
-        layer_config: &LayerConfig,
-        config: ContainerConfig,
+        target_config: &TargetConfig,
```

Comment:
why? how is this related?

Reply:
The pervious change kind-of breaks the operator, I'd rather create a dummy network config than the entirety of layer config

Reply:
Ok - I'd change it to optional, as it has no meaning if not for the later check.